frontend/index.html

```diff
@@ -37,6 +37,14 @@ <h1>Profiles</h1>
               <option value="default-docker">default-docker</option>
               <option value="weak-docker">weak-docker</option>
             </select>
+
+            <br><br>
+            SELinux:
+            <select name="selinux">
+                <option value="enabled">Enabled</option>
+                <option value="disabled">Disabled</option>
+            </select>
```

Comment:
Does not have to be in this PR, but it would be good if the Fedora VM only shows this while the Ubuntu VM only shows AppArmor. Maybe it's possible to pass a parameter to the provisioning script so that only one is shown?

Reply:
This can be done we generate this on the fly. Right now this is statically generated. Will start working on that.

Reply:
this is done.